### PR TITLE
[docs] Add quickstart tutorial to README #285

### LINKED ISSUE #285: [docs] Add quickstart tutorial to README
We need a quickstart that helps first time users to set up and understand the monitoring system.

Let's add more points to this issue!

- recommend setting openwisp-config `management_interface` option
- explain [how to trigger monitoring checks without management interface](https://github.com/openwisp/openwisp-monitoring#openwisp_monitoring_management_ip_only) (valid only if openwisp is in the same layer2 network)
- explain how default checks are created
- explain difference between active checks and passive metric collection
- explain that [openwrt-openwisp-monitoring](https://github.com/openwisp/openwrt-openwisp-monitoring/) is needed for some features to work (point out which features)

### PR DESCRIPTION
**Pull request by yashikajotwani12**
- [X] recommend setting openwisp-config `management_interface` option
- [ ] explain [how to trigger monitoring checks without management interface](https://github.com/openwisp/openwisp-monitoring#openwisp_monitoring_management_ip_only) (valid only if openwisp is in the same layer2 network)
- [ ]  explain how default checks are created
- [ ]  explain difference between active checks and passive metric collection
- [ ]  explain that [openwrt-openwisp-monitoring](https://github.com/openwisp/openwrt-openwisp-monitoring/) is needed for some features to work (point out which features)
* Fixes #285

Checks:

- [ ] I have manually tested the proposed changes
- [ ] I have written new test cases to avoid regressions (if necessary)
- [ ] I have updated the documentation (e.g. README.rst)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
                             gdal-bin libproj-dev libgeos-dev libspatialite-dev \
                             fping
 
+Quickstart
+----------
+
+1. Setting openwisp-config management_interface option
```
Comment (nemesisdesign) [inline README.rst:128]: make subheadings instead of lists, always flag code variables with double slant quotes.

File: README.rst [hunk 2]
```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
...
+
+* Make sure the management_interface setting of openwisp-config is set to the interface which is reachable by openwisp.
+* If the interface which is reachable is br-mng, add option management_interface 'br-mng' to `/etc/config/openwisp`.
```
Comment (nemesisdesign) [inline README.rst:131]: Don't use br-mng, use `tun0` which is more common.

File: README.rst [hunk 3]
```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
...
+
+.. code-block:: python
```
Comment (nemesisdesign) [inline README.rst:133]: this is not python!

File: README.rst [hunk 4]
```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
                             gdal-bin libproj-dev libgeos-dev libspatialite-dev \
                             fping
 
+Quickstart
+----------
+
+1. Setting openwisp-config management_interface option
+
+* Make sure the management_interface setting of openwisp-config is set to the interface which is reachable by openwisp.
```
Comment (nemesisdesign) [inline README.rst:130]: flag management_interface as variable please, remember restructured text needs double slant quotes:

```
``management_interface``
```

Applies to all the other occurrences.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Good progress! Can you complete all the points which are required? Please also look at the README changes in https://github.com/openwisp/openwisp-monitoring/pull/302 which introduce the monitoring package.

File: README.rst (hunk 4 above)
Comment (yashikajotwani12) [reply README.rst:130]: Surely would update every requested change soon. Thanks a lot, would follow these rules for the other points.

**Comment by pandafy** [comment]
Hi @yashikajotwani12! I will be taking over this PR because it is a release blocker (high priority). If you have made changes locally, kindly push them to update this PR. 
Thank you for contributing.

**Comment by yashikajotwani12** [comment]
> Hi @yashikajotwani12! I will be taking over this PR because it is a release blocker (high priority). If you have made changes locally, kindly push them to update this PR. Thank you for contributing.

Hello @pandafy, okay I understand.
I have updated it from my end.
Thanks.

**Comment by pandafy** [comment]
Superseded by https://github.com/openwisp/openwisp-monitoring/pull/302. 

Thank you @yashikajotwani12!

**Pull request closed without merging**